README.md

```diff
@@ -1,45 +1,60 @@
 # osb-checker
 An automatic checker to verify an Open Service Broker API implementation against the [specification](https://github.com/openservicebrokerapi/servicebroker).
 
-## Project Status
+# Project Status
 
 This project should be considered **experimental**. You should validate the results against the released [specification](https://github.com/openservicebrokerapi/servicebroker). In the case of any discrepancy, the specification should be considered correct.
 
-## Usage - test against the mock OSB API endpoint
+# Usage
 
-1. Clone the repository to a folder, for example c:\osb-checker
+## The Mock OSB API Server
+
+This repository comes with a mock OSB compatible server. Follow the below
+steps to start the mock server and run tests against it.
+
+1. Run `git clone` to clone this repository to `$DIRECTORY`
 2. Launch the mock server
 ```bash
-    cd c:\osb-checker\2.13\mocks
-    npm install
-    node mockOSB.js
+cd $DIRECTORY/2.13/mocks
```

Comment:
Do we want to include the latest version number in the readme? If we do we have to remember to update this on each release

Reply:
yep, good point. I'll change it to an env var